sisl/supercell.py

```diff
@@ -670,21 +670,20 @@ def _assert(m, v):
 
         return idx
 
-    def cell_vertices(self):
-        """Vertices of the cell.
+    def vertices(self):
+        """Vertices of the cell
         
         Returns
         --------
         array of shape (2, 2, 2, 3):
             The coordinates of the vertices of the cell. The first three dimensions
             correspond to each cell axis, and the last one contains the xyz coordinates.
         """
-        verts_fxyz = np.zeros((8,3))
-        verts_fxyz[4:, 0] = 1
-        verts_fxyz[[2, 3, 6, 7], 1] = 1
-        verts_fxyz[1::2, 2] = 1
-
-        return (verts_fxyz @ self.cell).reshape(2, 2, 2, 3)
+        verts = np.zeros([2, 2, 2, 3])
+        verts[1, :, :, 0] = 1
+        verts[:, 1, :, 1] = 1
+        verts[:, :, 1, 2] = 1
+        return (verts.reshape(-1, 3) @ self.cell).reshape(2, 2, 2, 3)
```

Comment:
I think there's no reason to reshape here, you can just go `return verts @ self.cell`

Reply:
true!